code/modules/clothing/head/jobs.dm

```diff
@@ -195,6 +195,6 @@
 	dog_fashion = /datum/dog_fashion/head/surgery
 
 /obj/item/clothing/head/surgery/black
-	desc = "A cap coroners wear during autopsies. Keeps their hair from falling into the cadavers.  It is as dark than the coroner's humor."
+	desc = "A cap coroners wear during autopsies. Keeps their hair from falling into the cadavers.  It is as dark as the coroners humor."
```

Comment:
If you're going to be fixing the grammar, please don't add another grammatical issue to the description. There should be an apostrophe in coroners -> coroner's   since it's a singular possessive noun.

Reply:
Whelp, that's embarrassing. I was looking at some other mistake and must've brain farted somewhere along the way. Fixed.